packages/form-handlers/src/mailchanels.ts

```diff
@@ -0,0 +1,84 @@
+// References:
+// https://api.mailchannels.net/tx/v1/documentation
+// https://github.com/cloudflare/pages-plugins/tree/main/packages/mailchannels
+//
+// To test see `packages/form-handlers-mailchannels-test`
+
+import { formDataToEmailContent } from "./shared";
+
+type EmailAddress = {
+  email: string;
+  name?: string;
+};
+
+type Personalization = {
+  to: [EmailAddress, ...EmailAddress[]];
+  from?: EmailAddress;
+  dkim_domain?: string;
+  dkim_private_key?: string;
+  dkim_selector?: string;
+  reply_to?: EmailAddress;
+  cc?: EmailAddress[];
+  bcc?: EmailAddress[];
+  subject?: string;
+  headers?: Record<string, string>;
+};
+
+type ContentItem = {
+  type: "text/plain" | "text/html";
+  value: string;
+};
+
+type SendEmailPayload = {
```

Comment:
Comment plz from where does this types same from. I mean API href, doc href etc

Reply:
There's comment at top of the file